src/main/java/com/github/cameltooling/lsp/internal/completion/CompletionResolveHandler.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cameltooling.lsp.internal.completion;
+
+import java.util.Map;
+
+import org.eclipse.lsp4j.CompletionItem;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+import org.eclipse.lsp4j.TextEdit;
+
+/**
+* @author lheinema
+*/
+public class CompletionResolveHandler {
+
+	public CompletionItem resolve(CompletionItem param) {
+		Map data = CompletionResolverUtils.getDataFromJSON(param.getData(), Map.class);
+		// clean resolve data
+		param.setData(null);
```

Comment:
what is the purpose of cleaning the data on the param?

Reply:
tbh I have no clue if its required. I copied that from another LSP.